pass-manager-svc/src/main/kotlin/com/example/passmanagersvc/web/mapper/proto/pass/CancelPassMapper.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.passmanagersvc.web.mapper.proto.pass
+
+import com.example.internal.commonmodels.Error
+import com.example.internal.input.reqreply.CancelPassResponse
+import com.example.passmanagersvc.exception.PassOwnerNotFoundException
+
+object CancelPassMapper {
+    fun successCancelPassResponse(): CancelPassResponse {
+        return CancelPassResponse.newBuilder().apply {
+            successBuilder
+        }.build()
+    }
+
+    fun failureCancelPassResponse(throwable: Throwable): CancelPassResponse {
+        return CancelPassResponse.newBuilder().also { responseBuilder ->
+            responseBuilder.failureBuilder.apply {
+                setMessage(throwable.message)
+                when (throwable) {
+                    is PassOwnerNotFoundException -> setPassOwnerNotFoundById(Error.getDefaultInstance())
+                }
+            }
+        }.build()
```

Comment:
```suggestion
        return CancelPassResponse.newBuilder().apply {
            failureBuilder.setMessage(throwable.message)
            when (throwable) {
                is PassOwnerNotFoundException -> failureBuilder.passOwnerNotFoundByIdBuilder
            }
        }.build()
```
use one scope function for better readability